libusb/os/emscripten_webusb.cpp

```diff
@@ -586,6 +586,11 @@ unsigned long getDeviceSessionId(val& web_usb_device) {
 }
 
 val getDeviceList(libusb_context* ctx, discovered_devs** devs) {
+	// Check if browser supports USB
+	val nav_usb = val::global("navigator")["usb"];
```

Comment:
Nit: since you already got this object anyway, can you reuse it where we need `navigator.usb` for the actual call below?

Also I'm a fan of longer names, they don't cost us anything, so I'd just call it `navigator_usb`.